kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/create-weblogic-sample-domain.sh

```diff
@@ -348,16 +365,66 @@ function createYamlFiles {
   sed -i -e "s:%ADMIN_NODE_PORT%:${adminNodePort}:g" ${dcrOutput}
   sed -i -e "s:%JAVA_OPTIONS%:${javaOptions}:g" ${dcrOutput}
   sed -i -e "s:%STARTUP_CONTROL%:${startupControl}:g" ${dcrOutput}
-
+ 
   # Remove any "...yaml-e" files left over from running sed
   rm -f ${domainOutputDir}/*.yaml-e
 }
 
+# create domain configmap using what is in the createDomainFilesDir
+function create_domain_configmap {
+  # Use the default files if createDomainFilesDir is not specified
+  if [ -z "${createDomainFilesDir}" ]; then
+    createDomainFilesDir=${scriptDir}/wlst
+  fi
+
+  # customize the files with domain information
+  local externalFilesTmpDir=$domainOutputDir/tmp
+  mkdir -p $externalFilesTmpDir
+  cp ${createDomainFilesDir}/* ${externalFilesTmpDir}/
+ 
+  for fname in ${externalFilesTmpDir}/*
+  do
+    local outputFile=$fname
+    sed -i -e "s:%NAMESPACE%:$namespace:g" ${outputFile}
+    sed -i -e "s:%WEBLOGIC_CREDENTIALS_SECRET_NAME%:${weblogicCredentialsSecretName}:g" ${outputFile}
```

Comment:
Should you match the operator helm chart?  It uses 'image', 'imagePullPolicy' and 'imagePullSecrets' (where the secret is the actual list of image pull secrets, instead of just a secret name, which would mean changing the same to parse yaml, instead of simple string values).

Reply:
I guess we should if we want to have a consistent/better way of doing the same thing. I'll take a look at the operator helm chart.

Reply:
I am changing the property names to be consistent with what we have in the operator helm charts:  weblogicImage -> image, and weblogicImagePullSecretName -> imagePullSecretName. Having discussed with Tom offine, we decided to leave the sample to only support one image pull secret.